CHANGELOG.rst

```diff
@@ -19,6 +20,7 @@ This document describes changes between each past release.
 - Upgrade kinto-admin to v1.24.1
 
 
+
```

Comment:
nitpick: Not sure this was necessary ;)

Reply:
Good catch. This was an accidental space. Removed.